cmd/manager/main.go

```diff
@@ -35,6 +35,14 @@ func printVersion() {
 	log.Info(fmt.Sprintf("operator-sdk Version: %v", sdkVersion.Version))
 }
 
+func getCleanVersion(version string) (string, error) {
+    reg, err := regexp.Compile("[^0-9]+")
```

Comment:
I'm more in favour of using directly a semantic versioning parser like `github.com/blang/semver`